thread/src/components/LoginForm.js

```diff
@@ -1,57 +1,82 @@
 import React, { Component } from 'react';
 import { Container, Content, Form, Item, Input, Label, Footer, FooterTab, Button, Text, Spinner } from 'native-base';
 import { connect } from 'react-redux';
+import {  View, StyleSheet } from 'react-native';
 import { Actions } from 'react-native-router-flux';
 import * as actions from '../actions';
-import { LoadingButton } from './Common';
+import styled from 'styled-components';
+import Logo from './Logo';
+import {LoadingButton, CommonContainer, CommonField} from './Common';
+
+const TermsButtonStyle = styled(Text)`
```

Comment:
Move this to common folder, as we need to reuse this in Register form.

Call it transparent button